public/doc/swagger-2-v0.0.1.yaml

```diff
@@ -1358,6 +1358,11 @@ definitions:
         type: string
         readOnly: true
         format: uuid
+      taggings:
```

Comment:
@agrare @bdunne do we prefer ```taggings``` or ```tag_list``` or something else? Relation is already named tags, so we can't use that.

Reply:
I guess taggings.

The more we talk about this, the more I think tags should be a subcollection

Reply:
Can we do subcollections so we don't have to do N+1 queries to retrieve VMs with tags?

Reply:
Not that I'm aware of

Reply:
Hm so subcollection will be good that we can paginate it, in a case that there's too much tags per entity (or we could limit it in this query maybe). But otherwise the usability will not be that good.

I think it'd be a good idea to start to design how the tag interface will look like (mainly filtering by tags). Then it could make it more clear what his should look like.

Reply:
We would not be able to paginate or limit this as an attribute, which we would be able to do with a subcollection.

As far as performance goes, we're limited by OpenApi / Swagger.

Either way, I think we should discuss the use cases first. cc @gtanzillo